workspaces/ui-v2/src/types/endpoints.ts

```diff
@@ -39,23 +41,31 @@ export interface IQueryParameters {
   rootShapeId: string;
   isRemoved: boolean;
   description: string;
+  endpointId: string;
+  pathId: string;
+  method: string;
 }
 
-export interface IRequestBody {
+export interface IRequest {
   requestId: string;
-  contentType: string;
-  rootShapeId: string;
+  description: string;
+  endpointId: string;
   pathId: string;
   method: string;
-  description: string;
+  body: IBody | null;
 }
 
-export interface IResponseBody {
+export interface IResponse {
   responseId: string;
   statusCode: number;
-  contentType: string;
-  rootShapeId: string;
+  endpointId: string;
   pathId: string;
   method: string;
   description: string;
+  body: IBody | null;
+}
```

Comment:
Currently what happens is, requests and responses created for every new content type (i.e. contentType `application/json` and `text/html` generate a new request/responseId - meaning that bodies is only _ever_ one or none.

I'm not 100% which one we'd want (i.e. a new request / response id per content type, or content types / bodies pointing to the same request node). Either way, this PR is big enough as it is, and should be updated in the spectacle projection -> into the UI